drifting-in-space/y-sweet
# try to fix types and tests

This makes the tests directory an npm workspace and switches to tsup for building our typescript projects.

To test this out:

1. Pull down this branch
2. Run `npm install` in the root or any workspace (will trigger an npm install for all workspaces)
3. Run `npm run dev` in `js-pkg/react` and/or `js-pkg/sdk`
4. Make some breaking changes
5. Run `npm test` from `tests`

Tests should fail.
